app/interactors/tasks/create.rb

```diff
@@ -2,7 +2,13 @@ module Tasks
   class Create
     include Interactor::Organizer
 
+    delegate :task, :project, to: :context
```

Comment:
And remove project argument in tasks_controller.rb
```
delegate :project, to: :task

==

def project
  @project ||= task.project
end
```
```suggestion
    delegate :task, to: :context
    delegate :project, to: :task
```